platform-sdk/consensus-otter-tests/src/testFixtures/java/org/hiero/otter/fixtures/logging/internal/InMemoryAppender.java

```diff
@@ -60,16 +62,29 @@ protected InMemoryAppender(@NonNull final String name) {
      */
     @Override
     public void append(@NonNull final LogEvent event) {
+        final long nodeId = convertSafelyToLong(event.getContextData().getValue(TurtleNode.THREAD_CONTEXT_NODE_ID));
```

Comment:
Does this mean we display `-1` if no nodeId is set?

Reply:
added for the toString a check to write "unknown"